packages/opentelemetry-sdk-trace-base/src/export/ReadableSpan.ts

```diff
@@ -15,29 +15,30 @@
  */
 
 import {
-  SpanKind,
-  SpanStatus,
   Attributes,
   HrTime,
   Link,
   SpanContext,
+  SpanKind,
+  SpanStatus,
 } from '@opentelemetry/api';
-import { Resource } from '@opentelemetry/resources';
 import { InstrumentationScope } from '@opentelemetry/core';
+import { Resource } from '@opentelemetry/resources';
 import { TimedEvent } from '../TimedEvent';
 
 export interface ReadableSpan {
   readonly name: string;
   readonly kind: SpanKind;
   readonly spanContext: () => SpanContext;
   readonly parentSpanContext?: SpanContext;
+  readonly startTimeUnixNano: bigint;
+  readonly endTimeUnixNano: bigint;
   readonly startTime: HrTime;
   readonly endTime: HrTime;
   readonly status: SpanStatus;
   readonly attributes: Attributes;
   readonly links: Link[];
   readonly events: TimedEvent[];
-  readonly duration: HrTime;
```

Comment:
If we decide to keep compatibility, this should be kept. Checked on the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#additional-span-interfaces), `duration` is not required by it and we may not need to add a nanos equivalent.